inputs/values/resources_hg38.json

```
@@ -40,65 +40,20 @@
-  "ssc_belyeu_benchmark_calls" : [
-    "gs://talkowski-sv-gnomad-resources/resources/hg38_benchmarking/SSC_Belyeu/SSC_Belyeu.SV.AFR.bed.gz",
-    "gs://talkowski-sv-gnomad-resources/resources/hg38_benchmarking/SSC_Belyeu/SSC_Belyeu.SV.ALL.bed.gz",
-    "gs://talkowski-sv-gnomad-resources/resources/hg38_benchmarking/SSC_Belyeu/SSC_Belyeu.SV.AMR.bed.gz",
-    "gs://talkowski-sv-gnomad-resources/resources/hg38_benchmarking/SSC_Belyeu/SSC_Belyeu.SV.EAS.bed.gz",
-    "gs://talkowski-sv-gnomad-resources/resources/hg38_benchmarking/SSC_Belyeu/SSC_Belyeu.SV.EUR.bed.gz"
-  ],
-  "ssc_sanders_benchmark_tarball": "gs://talkowski-sv-gnomad-resources/resources/hg38_benchmarking/SSC_Sanders_perSample.v2.tar.gz",
-  "thousand_genomes_benchmark_calls" : [
-    "gs://gatk-sv-resources-public/hg38/v0/sv-resources/resources/v1/1000G_Sudmant/1000G_Sudmant.SV.AFR.bed.gz",
-    "gs://gatk-sv-resources-public/hg38/v0/sv-resources/resources/v1/1000G_Sudmant/1000G_Sudmant.SV.ALL.bed.gz",
-    "gs://gatk-sv-resources-public/hg38/v0/sv-resources/resources/v1/1000G_Sudmant/1000G_Sudmant.SV.AMR.bed.gz",
-    "gs://gatk-sv-resources-public/hg38/v0/sv-resources/resources/v1/1000G_Sudmant/1000G_Sudmant.SV.EAS.bed.gz",
-    "gs://gatk-sv-resources-public/hg38/v0/sv-resources/resources/v1/1000G_Sudmant/1000G_Sudmant.SV.EUR.bed.gz",
-    "gs://gatk-sv-resources-public/hg38/v0/sv-resources/resources/v1/1000G_Sudmant/1000G_Sudmant.SV.SAS.bed.gz"
-  ],
+  "ccdg_abel_site_level_benchmarking_dataset" : ["CCDG_abel", "gs://gatk-sv-resources-public/hg38/v0/sv-resources/resources/v1/CCDG_Abel"],
+  "gnomad_v2_collins_sample_level_benchmarking_dataset": ["gnomAD_v2_Collins", "gs://gatk-sv-resources-secure/hg38/v0/sv-resources/resources/v1/gnomAD_v2_Collins_perSample.tar.gz"],
+  "gnomad_v2_collins_site_level_benchmarking_dataset" : ["gnomAD_v2_Collins", "gs://gatk-sv-resources-public/hg38/v0/sv-resources/resources/v1/gnomAD_v2_Collins"],
+  "hgsv_byrska_bishop_sample_level_benchmarking_dataset": ["HGSV_ByrskaBishop", "gs://gatk-sv-resources-public/hg38/v0/sv-resources/resources/v1/HGSV_ByrskaBishop_GATKSV_perSample.tar.gz"],
+  "hgsv_byrska_bishop_site_level_benchmarking_dataset" : ["HGSV_ByrskaBishop", "gs://gatk-sv-resources-public/hg38/v0/sv-resources/resources/v1/HGSV_ByrskaBishop_GATKSV"],
+  "hgsv_ebert_sample_level_benchmarking_dataset": ["HGSV_Ebert", "gs://gatk-sv-resources/resources/hg38_benchmarking/HGSV_Ebert_perSample.tar.gz"],
+  "ssc_belyeu_sample_level_benchmarking_dataset": ["SSC_Belyeu", "gs://talkowski-sv-gnomad-resources/resources/hg38_benchmarking/SSC_Belyeu_perSample.tar.gz"],
+  "ssc_belyeu_site_level_benchmarking_dataset" : ["SSC_Belyeu", "gs://talkowski-sv-gnomad-resources/resources/hg38_benchmarking/SSC_Belyeu"],
+  "ssc_sanders_sample_level_benchmarking_dataset": ["SSC_Sanders", "gs://talkowski-sv-gnomad-resources/resources/hg38_benchmarking/SSC_Sanders_perSample.v2.tar.gz"],
```

```suggestion
  "ssc_belyeu_site_level_benchmarking_dataset" : ["SSC_Belyeu", "gs://gatk-sv-resources-secure/resources/hg38_benchmarking/SSC_Belyeu"],
  "ssc_sanders_sample_level_benchmarking_dataset": ["SSC_Sanders", "gs://gatk-sv-resources-secure/resources/hg38_benchmarking/SSC_Sanders_perSample.v2.tar.gz"],
```
I started moving these over. Maybe move up the other files from `/resources` into `/resources/hg38_benchmarking` also?

---

I copied Collins, Werling, and Sanders from `/resources` into `/resources/hg38_benchmarking` - did not yet delete the copy in `/resources`